reference/api/settings.md

```diff
@@ -159,34 +166,230 @@ If the provided index does not exist, it will be created.
-| **filterableAttributes** | [Strings] | Attributes to use as [filters and facets](/learn/advanced/filtering_and_faceted_search.md) | `[]`                                                                                         |
-| **pagination**           | Object    | Pagination settings                                                                        | `{}`                                                                                         |
-| **rankingRules**         | [Strings] | List of ranking rules sorted by order of importance                                        | [A list of ordered built-in ranking rules](/learn/core_concepts/relevancy.md#built-in-rules) |
-| **searchableAttributes** | [Strings] | Fields in which to search for matching query words sorted by order of importance           | `["*"]` (all attributes)                                                                     |
-| **sortableAttributes**   | [Strings] | Attributes to use when [sorting](/learn/advanced/sorting.md) search results                | `[]`                                                                                         |
-| **stopWords**            | [Strings] | List of words ignored by Meilisearch when present in search queries                        | `[]`                                                                                         |
-| **synonyms**             | Object    | List of associated words treated similarly                                                 | `{}`                                                                                         |
-| **typoTolerance**        | Object    | Typo tolerance settings                                                                    | `{}`                                                                                         |
-[Learn more about the settings](/learn/configuration/settings.md).
+| Name        | Type   | Description                                                               |
+| :---------- | :----- | :------------------------------------------------------------------------ |
+| **`uid`** * | String | [`uid`](/learn/core_concepts/indexes.md#index-uid) of the requested index |
 
 #### Example
 
 <CodeSamples id="reset_settings_1" />
 
-#### Response: `202 Accepted`
+##### Response: `202 Accepted`
+
+```json
+{
+    "taskUid": 1,
+    "indexUid": "movies",
+    "status": "enqueued",
+    "type": "settingsUpdate",
+    "enqueuedAt": "2021-08-11T09:25:53.000000Z"
+}
+```
+
+You can use this `taskUid` to get more details on [the status of the task](/reference/api/tasks.md#get-one-task).
+
+## Displayed attributes
+
+The fields whose attributes are added to the displayed attributes list are displayed in each matching document.
+
+By default, all fields are considered to be `displayedAttributes`. This behavior is represented by the `*` in the settings. Setting `displayedAttributes` to an empty array `[]` will reset the setting to its default value.
```

Comment:
I think this paragraph should be moved or removed. Consider similar settings like `searchableAttributes`, which also has default value `["*"]`. Where do we mention this information for that setting? If we don't, where do you think would be the best place to mention this for both settings?

```suggestion
```

Reply:
I think it would be better to add this for `searchableAttributes` too. Users will probably be looking at each setting separately